osf/models/user.py

```diff
@@ -1785,6 +1789,29 @@ def gdpr_delete(self):
         self.external_identity = {}
         self.deleted = timezone.now()
 
+    @property
+    def has_resources(self):
+        """
+        This is meant to determine if a user has any resources, nodes, preprints etc that might impede their deactivation.
+        If a user only has no resources or only deleted resources this will return false and they can safely be deactivated
+        otherwise they must delete or transfer their outstanding resources.
+
+        :return bool: does the user have any active node, preprints, groups, quickfiles etc?
+        """
+        from osf.models import Preprint
+
+        # TODO: Update once quickfolders in merged
+
+        nodes = self.nodes.exclude(type='osf.quickfilesnode', is_deleted=True).exists()
```

Comment:
You want to return non-deleted files that are not quickfiles - so this instead? As it is currently, you're returning a queryset of all nodes as long as they are not deleting quickfile nodes.
this instead?
```
self.nodes.exclude(type='osf.quickfilesnode').filter(is_deleted=True)
```